Jenkinsfile

```diff
@@ -27,9 +27,24 @@ pipeline {
         }
     }
 
-    stage('Image Build') {
-      steps {
-        sh './bin/build'
+    stage('Build') {
+      parallel {
+        stage('Build Images') {
+          steps {
+            sh './bin/build'
+          }
+        }
+
+        stage('Build Release Artifacts') {
+          when {
+            branch 'master'
+          }
+
+          steps {
+            sh './bin/build_release --snapshot'
+            archiveArtifacts 'dist/goreleaser/'
+          }
+        }
```

Comment:
Building release artifacts on each build step before even running tests seems like an increase to the build time with no benefits to the developer for being done this early. If we really want the artifacts saved, we should probably do it when the tests all pass.

Reply:
Good point. I can move this particular stage to after the tests